CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+## 12.2.1
+
+_released `2023-01-18`_
+
+### cli
+
+- :scroll: Improve docs about input field types ([#585](https://github.com/zapier/zapier-platform/pull/585))
+- :scroll: Improve docs about OAuth2 fields ([#589](https://github.com/zapier/zapier-platform/pull/589))
+- :scroll: Update logging info ([#591](https://github.com/zapier/zapier-platform/pull/591))
+- :scroll: Fix `zapier init` command ([#594](https://github.com/zapier/zapier-platform/pull/594))
+- :scroll: Fix typo ([#593](https://github.com/zapier/zapier-platform/pull/593))
+- :scroll: Fix logo ([#604](https://github.com/zapier/zapier-platform/pull/604))
+- :hammer: Dependency updates
+  - Bump typescript from 3.8.3 to 4.9.4 to fix a test ([#599](https://github.com/zapier/zapier-platform/pull/599))
+  - Bump decode-uri-component from 0.2.0 to 0.2.2 ([#596](https://github.com/zapier/zapier-platform/pull/596))
+  - Bump json5 from 1.0.1 to 1.0.2 ([#601](https://github.com/zapier/zapier-platform/pull/601))
```

Comment:
I guess you intentionally left out these but I thought I should confirm anyways.

- https://github.com/zapier/zapier-platform/pull/590
- https://github.com/zapier/zapier-platform/pull/600

Reply:
I intentionally left out #600 because it belongs to legacy-scripting-runner's changelog. I missed #590. Now I've included it. Good catch!